### PR TITLE
Implement deployment commands

### PR DESCRIPTION
**Pull request by matthiashamacher**
Add necessary deployment commands to the deploy step

### CODE REVIEW

File: .platform.app.yaml [hunk 1]
```diff
@@ -251,6 +251,47 @@ hooks:
 
     # Clear env cached
     ./bin/console cache:clear
+      
+    # get all bundles which are enabled and installable but not yet installed
+    BUNDLES="$(
+      bin/console pimcore:bundle:list --json \
+        | jq --raw-output '
```
Comment (matthiashamacher) [inline .platform.app.yaml:258]: @gilzow Is `jq` already installed or do we need to add it?

Comment (gilzow) [reply .platform.app.yaml:258]: yep, `jq` is already installed so no need to worry about it.

File: .platform.app.yaml [hunk 2]
```diff
@@ -251,6 +251,47 @@ hooks:
...
+    ./bin/console doctrine:migrations:migrate --no-interaction --allow-no-migration
+      
+    # Rebuild class definitions
+    ./bin/console pimcore:deployment:classes-rebuild --create-classes --delete-classes --no-interaction
+      
+    # Rebuild custom layouts
+    ./bin/console pimcore:deployment:custom-layouts-rebuild --create-custom-layouts --delete-custom-layouts --no-interaction
+      
+    # Dump autoload to update classmap with new classes of pimcore
+    composer dump-autoload --optimize --apcu --no-dev
```
Comment (gilzow) [inline .platform.app.yaml]: Unfortunately, we can't run this command in the deploy hook as the application container is read-only at this point.
```
    W:   file_put_contents(/app/vendor/composer/autoload_namespaces.php): Failed to   
    W:   open stream: Read-only file system  
```
Any reason we can't move this to the end of the build hook?

File: .platform.app.yaml [hunk 3]
```diff
@@ -251,6 +251,47 @@ hooks:
...
+    )"
+  
+    # Install Pimcore bundles
+    for BUNDLE in ${BUNDLES}; do
+      bin/console pimcore:bundle:install "${BUNDLE}" --no-interaction --no-post-change-commands
+    done
+  
+  
+    # Migrate doctrine migrations
+    ./bin/console doctrine:migrations:migrate --no-interaction --allow-no-migration
```
Comment (gilzow) [inline .platform.app.yaml:279]: This failed on `Migration Pimcore\Bundle\CoreBundle\Migrations\Version20210531125102` with 
```
Column already exists: 1060 Duplicate column name 'localizedErrorDocuments'" 3:17:04 AM 45
```
Is this normal or a cause for concern?

**Pull request merged**